benchto-service/src/main/java/io/trino/benchto/service/model/Environment.java

```diff
@@ -62,7 +62,7 @@
     @JsonIgnore
     private long id;
 
-    @Size(min = 1, max = 64)
+    @Size(min = 1, max = 96)
```

Comment:
would this require database migration? see https://github.com/trinodb/benchto/tree/master/benchto-service/src/main/resources/db/migration

Reply:
Yes, forgot to add this as I run manually on our DB.
Anyway added now.